tests/unit/Deserializers/SnakDeserializerTest.php

```diff
@@ -113,6 +115,14 @@ public function deserializationProvider() {
 					'hash' => 'not a valid hash'
 				]
 			],
+			[
+				new PropertyNoValueSnak( new PropertyId( 'foo:P42' ) ),
```

Comment:
This line here makes the dataProvider fail. The code you want to test is not reached.

Reply:
Works for DataModel version 6, but not before. Meh.

Reply:
Ah, yea. We could drop this compatibility requirement from travis.yml. Would be fine for me.

Reply:
If you drop it from travis.yml, also drop it from composer.json.
I removed the test for now, it's not really necessary.